am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/RolePermissions.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.amlib.enums.AccessType;
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+
+import java.util.Map;
+import java.util.Set;
+
+@Data
+@Builder
+public final class RolePermissions {
+    private final Map<JsonPointer, Set<Permission>> permissions;
+    private final AccessType accessType;
```

Comment:
We said during planning to name it `roleAccessType`. 

This way we will have:

```
RolePermissions:
  permissions:
  securityClassifications:
  roleSecurityClassification: 
  roleAccessType:
```

I also find above order of properties bit more tidy :)